### ISSUE TITLE
Could this eventually support private subreddits & praw / authentication?

### ISSUE THREAD

**Issue by github-userx** [libertysoft3/reddit-html-archiver#6]

**Comment by libertysoft3** [comment]
I'm pretty sure that private subs are supported now, the data comes from PushShift.

Yeah, praw could be added somewhat easily, depending on what you're trying to do with it. It would be cool to have praw fetch updates scores.

**Comment by github-userx** [comment]
Huh? Since when does pushshift archive private subreddits? Now I am so confused. Pushshift doesn’t have any access to private subreddits AFAIK!

31. Mai 2019, 05:03 from notifications@github.com:

**Comment by libertysoft3** [comment]
Yeah I was full of it and tried to fix it with an edit. They might have some private sub data in one scenario only. Anyway what about the 1000 post limit with the reddit API?

**Comment by github-userx** [comment]
I would already be happy to get 1000 posts from a private subreddit

31. Mai 2019, 09:52 from notifications@github.com:

**Comment by libertysoft3** [comment]
I'm into it, pull requests accepted. Everyone that wants this please give this issue a thumbs up.

**Comment by github-userx** [comment]
Any update?

**Comment by libertysoft3** [comment]
I'm pretty busy with SaidIt lately.. I don't have this scheduled to be done currently.

**Comment by github-userx** [comment]
Ok, thanks! No worries! ;)

**Comment by saberman888** [comment]
Hey I tried looking into adding my own implementation, but it looks like this isn't possible anymore because of Reddit's deprecation of cloudsearch. So I can't search posts by any specific date under this newer search system, and looking at r/changelog there hasn't been any updates to this newer system since then. Let me know If I missed anything

https://www.reddit.com/r/changelog/comments/7tus5f/update_to_search_api/

**Comment by github-userx** [comment]
Thanks a lot! Really sucks how they changed the API. ;(

**Comment by libertysoft3** [comment]
Yep, PRAW talks about it in their changelog for version 6.0.0:

> Removed: Subreddit.submissions as the API endpoint backing the method is no more. See https://www.reddit.com/r/changelog/comments/7tus5f/update_to_search_api/.

https://praw.readthedocs.io/en/latest/package_info/change_log.html

So I guess the best that can be done for private subs is constantly consuming new posts, and getting 1000 posts out of each different API sort that's offered.
